src/rplidar_nodelet.cpp

```diff
@@ -152,8 +155,9 @@ namespace rplidar_ros {
     start_scan_time = ros::Time::now();
     op_result = drv->grabScanData(nodes, count);
     end_scan_time = ros::Time::now();
-    scan_duration = (end_scan_time - start_scan_time).toSec() * 1e-3;
-
+    scan_duration = (end_scan_time - start_scan_time).toSec();// * 1e-3;
```

Comment:
When I did the timing, I timed the whole loop into 3 different parts: data acquisition from serial, transformation to ROS LaserScan and publishing, and the total (which should be the sum). Without this change, times will not match. This value is later assigned to scan_time, which is "time between scans [s]". In fact, I am still not sure if this is the correct time, as it does not measure exactly time between scans. But, in any case, I think it was a bug.